_topic_maps/_topic_map.yml

```diff
@@ -3110,6 +3110,8 @@ Topics:
     File: virt-installing-virtio-drivers-on-existing-windows-vm
   - Name: Installing VirtIO driver on a new Windows virtual machine
     File: virt-installing-virtio-drivers-on-new-windows-vm
+  - Name: vTPM devices
```

Comment:
I'd add a gerund to the beginning of the title ([as mentioned in the contrib guide](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings)) and write out `vTPM` to match the title of the assembly. A quick search leads me to believe that Trusted Platform Module should be capitalized as well. I'd suggest something like `Using virtual Trusted Platform Module devices`